### ISSUE TITLE
Hi @johnduffell,  in present, does your app public as open source software?

### ISSUE THREAD

**Issue by goonux** [guardian/frontend#21857]
Hi @johnduffell,  in present, does your app public as open source software?
I want try to install the front-end on my local computer but it doesn't work.
![guardian-error](https://user-images.githubusercontent.com/8654203/65417558-87338400-de24-11e9-8be3-70179e5d03b8.png)
Could you take a look at this and give me an advice?

_Originally posted by @goonux in https://github.com/guardian/frontend/issues/13691#issuecomment-534036741_

**Comment by goonux** [comment]
I mean that there are have any ways to deploy the front-end on my local machine?

**Comment by johnduffell** [comment]
no, sorry, as before we are working in the open so you can see and review all our code, but the configuration and other specific data is stored privately in our AWS account.  If you can work out some configuration values and get access to our content api https://open-platform.theguardian.com/ you can get it up and running against your own AWS account for articles, but there are still many other services you would need.  I am not sure that this will change soon unfortunately.

**Comment by goonux** [comment]
I see, thanks for your explanation, so is there has anyway that can help people like me can get over the installation easier? 
like as an installation document or I have to walk through it by myself  :)

**Comment by johnduffell** [comment]
Sorry, there's nothing as far as I know.  Also, it has never been tested as a normal usage, so I wonder if the limitations would be too restrictive to do much of use.
But feel free to write a document yourself if you are motivated to analyse things more, and it would make a good blog post!  I'm not in that team, but they might be happy to link to a document if you manage to solve it!

**Comment by oliverlloyd** [comment]
Hi @goonux 👋 Thanks for your interest. If you like, you can also try [DCR](https://github.com/guardian/dotcom-rendering), this is the new version of the theguardian.com, running it locally is a lot easier and doesn't require any special credentials - most people have it up and running in a few minutes 😎
